pyproject.toml

```diff
@@ -64,39 +64,36 @@ module = [
   "sparse.*",
   "toolz.*",
   "zarr.*",
-  "numpy.exceptions.*",  # remove once support for `numpy<2.0` has been dropped
+  "numpy.exceptions.*", # remove once support for `numpy<2.0` has been dropped
 ]
 
 [[tool.mypy.overrides]]
 ignore_errors = true
 module = []
 
 [tool.ruff]
-target-version = "py39"
 builtins = ["ellipsis"]
 exclude = [
-    ".eggs",
-    "doc",
-    "_typed_ops.pyi",
+  ".eggs",
+  "doc",
+  "_typed_ops.pyi",
 ]
+target-version = "py39"
 # E402: module level import not at top of file
 # E501: line too long - let black worry about that
 # E731: do not assign a lambda expression, use a def
 ignore = [
-    "E402",
-    "E501",
-    "E731",
+  "E402",
+  "E501",
+  "E731",
 ]
 select = [
-    # Pyflakes
-    "F",
-    # Pycodestyle
-    "E",
-    "W",
-    # isort
-    "I",
-    # Pyupgrade
-    "UP",
+  # Pyflakes
```

Comment:
The comments are all one line too high

Reply:
Thanks, sorry I prob should have reverted the autoformatter. But now fixed

Reply:
Isort and pyupgrade are still one too high ;)

Reply:
D'oh! Thanks. #8185